src/components/Calculator.js

```diff
@@ -1,41 +1,54 @@
-
-        <button name="7" type="button" className="button">7</button>
-        <button name="8" type="button" className="button">8</button>
-        <button name="9" type="button" className="button">9</button>
-        <button name="*" type="button" className="operators">×</button>
-
-        <button name="4" type="button" className="button">4</button>
-        <button name="5" type="button" className="button">5</button>
-        <button name="6" type="button" className="button">6</button>
-        <button name="-" type="button" className="operators">-</button>
-
-        <button name="1" type="button" className="button">1</button>
-        <button name="2" type="button" className="button">2</button>
-        <button name="3" type="button" className="button">3</button>
-        <button name="+" type="button" className="operators">+</button>
-
-        <button name="0" type="button" id="zero" className="button">0</button>
-        <button name="." type="button" className="button">.</button>
-        <button name="=" type="button" className="operators">=</button>
+        <Result className="result" result={next || (total || '0')} />
+        <button onClick={this.Handler} name="AC" type="button" className="button">AC</button>
+        <button onClick={this.Handler} name="+/-" type="button" className="button">+/-</button>
+        <button onClick={this.Handler} name="%" type="button" className="button">%</button>
+        <button onClick={this.Handler} name="/" type="button" className="operators">÷</button>
+
+        <button onClick={this.Handler} name="7" type="button" className="button">7</button>
+        <button onClick={this.Handler} name="8" type="button" className="button">8</button>
+        <button onClick={this.Handler} name="9" type="button" className="button">9</button>
+        <button onClick={this.Handler} name="x" type="button" className="operators">x</button>
+
+        <button onClick={this.Handler} name="4" type="button" className="button">4</button>
+        <button onClick={this.Handler} name="5" type="button" className="button">5</button>
+        <button onClick={this.Handler} name="6" type="button" className="button">6</button>
+        <button onClick={this.Handler} name="-" type="button" className="operators">-</button>
+
+        <button onClick={this.Handler} name="1" type="button" className="button">1</button>
+        <button onClick={this.Handler} name="2" type="button" className="button">2</button>
+        <button onClick={this.Handler} name="3" type="button" className="button">3</button>
+        <button onClick={this.Handler} name="+" type="button" className="operators">+</button>
+
+        <button onClick={this.Handler} name="0" type="button" id="zero" className="button">0</button>
+        <button onClick={this.Handler} name="." type="button" className="button">.</button>
+        <button onClick={this.Handler} name="=" type="button" className="operators">=</button>
```

Comment:
- [ ] [OPTIONAL] There is a way to avoid all these `onClick` listeners and use only one, this technique is called `bubbling up` 😄 
    - I will leave here a link to a [youtube tutorial](https://www.youtube.com/watch?v=SqQZ8SttQsI),  this could be helpful in this project and in future projects 🚀